### PR TITLE
Complete my_object - SassyOven

### PR DESCRIPTION
**Pull request by timsjpark**
[#105361970]

### CODE REVIEW

File: objects/my_object.rb [hunk 1]
```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env ruby
```
Comment (jasonnoble) [inline objects/my_object.rb]: Don't need this to be executable...

File: objects/my_object.rb [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+require_relative 'my_object_auxiliary_methods'
+
+# BAKE ALL THE THINGS!!!
+class SassyOven
```
Comment (jasonnoble) [inline objects/my_object.rb]: I would expect to find the `SassyOven` class in a file named `objects/sassy_oven.rb`.

File: objects/my_object.rb [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  def initialize
+    @temp = 80
+    @filth = 0
+    @door = 'close'
```
Comment (jasonnoble) [inline objects/my_object.rb]: Maybe `@door_closed` boolean?

File: objects/my_object.rb [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+  def broil
+    heat_up(500, :fast)
+    bake
+  end
+
+  # This allows you to bake food. This method should make sure that the door
+  # is open first and then use the bake_process method to cook the food.
+
+  def bake
+    # Leave me alone Rubocop. I know what I'm doing.
```
Comment (jasonnoble) [inline objects/my_object.rb]: "I know what I'm doing" he says...  LOL.

52399_laughing-despicable-me-minions

File: objects/my_object.rb (hunk 3 above)
Comment (timsjpark) [inline objects/my_object.rb]: I thought about this and it was easier to follow what was happening if I used a string that represented it's exact state.

**Pull request merged**